src/ert/__main__.py

```diff
@@ -50,8 +49,8 @@ def run_ert_storage(args):
 
 def run_webviz_ert(args):
     try:
-        # pylint: disable=unused-import
-        import webviz_ert  # noqa
+        # pylint: disable=unused-import,import-outside-toplevel
```

Comment:
Why are the new pylint exceptions in this file now required?

Reply:
so they are not necessary - indeed in our github action, we ignore `src/ert/__main__.py`.
i added them because i run pylint locally, preemptively, to not have it fail in CI. perhaps i should add the ignores we use in CI...
should i remove the pylint inline ignores in files we ignore in CI?

Reply:
You can keep them I was just wondering why they poped up in your PR